### PR TITLE
Adding composer autoload, fixing unit tests

### PR DESCRIPTION
**Pull request by srtfisher**
- Composer autoloader is being used to load wp-asset-manager not as a plugin (for mantle)
- Removes travis and changes to github actions.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          command: composer install
+
+      - name: Install WordPress and Copy Project
+        shell: bash
+        run: |
+          bash bin/install-wp-tests.sh wordpress_unit_tests root '' 127.0.0.1 $WP_VERSION
+          composer global require "phpunit/phpunit=4.8.*|5.7.*"
+
+          # Go into the core directory and replace wp-content.
+          # mkdir -p ${WP_CORE_DIR}/wp-content/plugins/wp-asset-manager
```
Comment (kjbenk) [inline .github/workflows/test.yml]: Is this line needed anymore?

File: asset-manager.php [hunk 2]
```diff
@@ -26,13 +26,15 @@
...
 define( 'AM_BASE_DIR', dirname( __FILE__ ) );
 
-/**
- * Load base classes
- */
-require_once AM_BASE_DIR . '/php/class-asset-manager.php';
-require_once AM_BASE_DIR . '/php/class-asset-manager-scripts.php';
-require_once AM_BASE_DIR . '/php/class-asset-manager-styles.php';
-require_once AM_BASE_DIR . '/php/class-asset-manager-preload.php';
+if ( ! class_exists( 'Asset_Manager' ) ) :
```
Comment (kjbenk) [inline asset-manager.php:29]: Out of curiosity, why is this check needed?

**Review by kjbenk — APPROVED** [review]
⛳

File: asset-manager.php (hunk 2 above)
Comment (srtfisher) [reply asset-manager.php:29]: In the weird edge case that it's loaded with composer and then somehow activated through the plugin -- it's probably not needed in hindsight

**Pull request merged**